docs/integrations/azureBlobstorage/discovery.md

```diff
@@ -0,0 +1,87 @@
+---
+id: discovery
+title: Azure BlobStorage Discovery
+sidebar_label: Discovery
+# prettier-ignore
+description: Automatically discovering catalog entities from an Azure BlobStorage container
+---
+
+The Azure BlobStorage integration has a special entity provider for discovering
+catalog entities within an Azure BlobStorage. The provider will crawl your provided Azure
+BlobStorage container and register entities matching the configured path. This can
+be useful as an alternative to static locations or manually adding things to the
+catalog.
+
+This guide explains how to install and configure the Azure BlobStorage Entity Provider.
+
+## Dependencies
+
+### Azure Integration
+
+Setup [Azure integration](locations.md) with `accountName` and `secretAccessKey`. Host will than
+be `{accountName}.blob.core.windows.net`.
```

Comment:
Does the configuration support national clouds (e.g. China, US Gov) which have different url suffixes?  If so, would be great to add how to do this to the docs.  If not, I think it's ok to call out that they're not supported.